applications_tests/gls_sharp_navier_stokes_3d/coupled_moving_stokes.prm

```diff
@@ -0,0 +1,329 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf2
+  set time step               = 0.125
+  set number mesh adapt       = 2       # If steady, nb mesh adaptation
+  set time end                = 0.125 # End time of simulation
+  set output path             =  # Path to where the output results are made
```

Comment:
if you don't want to specify output path, wouldn't it be better to remove this line